src/content/docs/telemetry-data-platform/manage-data/nrintegrationerror.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Use NrIntegrationError event to understand data ingest problems
+metaDescription: Use the NrIntegrationError event to understand problems with ingest, integration with New Relic, and limits. 
+---
+
+Use New Relic's `NrIntegrationError` event to understand problems related to data ingest and feature configuration.  
+
+## What is NrIntegrationError used for? [#overview]
+
+Our `NrIntegrationError` event is used to record error messages related to: 
+
+* Exceeding data limits 
+* Malformed data
+* Configuration issues
+
+You can run queries to analyze the errors that are being generated. You can also set up alerts to be notified when specific types of errors, or quantities of errors, occur. 
+
+The primary way `NrIntegrationError` events are categorized is by the `newRelicFeature` attribute. `newRelicFeature` values include: 
```

Comment:
to reduce passive: Categorize `NrIntegrationError` events primarily using the `newRelicFeature` attribute.